tests/integration_tests/style/test_licenses.py

```diff
@@ -2,9 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0
 """Tests checking against the existence of licenses in each file."""
 
+from datetime import datetime
 import framework.utils as utils
 
-AMAZON_COPYRIGHT_YEARS = (2018, 2019, 2020)
+AMAZON_COPYRIGHT_YEARS = range(2018, datetime.now().year + 1)
```

Comment:
Isn't this addressed in https://github.com/firecracker-microvm/firecracker/pull/2423/commits/fe26cb2c90a1923dce645939094ee3fcffc4380e ?

Reply:
it is. It's just a race for the rebase :))